check_nightly_success/check-nightly-success/check.py

```diff
@@ -0,0 +1,74 @@
+import requests
```

Comment:
Should we set up basic Python linting in this repo? Just a Ruff pre-commit config with formatting and E, F, W, I rules would suffice.

Reply:
Done.